src/bootstrap/bin/main.rs

```diff
@@ -5,11 +5,40 @@
 //! parent directory, and otherwise documentation can be found throughout the `build`
 //! directory in each respective module.
 
-use std::env;
+use std::{collections::HashSet, env};
 
 use bootstrap::{Build, Config, Subcommand, VERSION};
 
 fn main() {
+    if cfg!(windows) {
```

Comment:
I think only the Windows terminal change needs to be gated to Windows. Removing duplicates seems useful in general.